### PR TITLE
Fixed bug with invoke cli and invoke nbshell.

### PR DESCRIPTION
**Pull request by gsnider2195**
# Closes: #DNE
# What's Changed

After #5055 when running `invoke cli` or `invoke nbshell` you get this error:

`TypeError: tasks.docker_compose() got multiple values for keyword argument 'pty'`

This fixes that bug.

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example Plugin Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -436,7 +436,7 @@ def nbshell(context):
     """Launch an interactive Nautobot shell."""
     command = "nautobot-server nbshell"
 
-    run_command(context, command, pty=True)
+    run_command(context, command)
```
Comment (glennmatthews) [inline tasks.py:439]: I'd fixed this a different way locally, by changing `run_command`:

```diff
-        return docker_compose(context, compose_command, pty=True, **kwargs)
+        kwargs["pty"] = True
+        return docker_compose(context, compose_command, **kwargs)
```

Thoughts on one versus the other?

File: changes/5079.fixed
Comment (glennmatthews) [inline changes/5079.fixed:1]: Probably should be `.housekeeping`?

File: tasks.py (hunk 1 above)
Comment (gsnider2195) [reply tasks.py:439]: I don't think it matters either way

**Review by HanlinMiao — APPROVED** [review]

**Comment by HanlinMiao** [comment]
Seems to be broken in `next` as well, should we port this to `next`?

**Pull request merged**